manifests/datadog.pp

```diff
@@ -0,0 +1,12 @@
+#
+# === Class: profile::datadog
+#
+# Setup datadog agent
+#
+# === Parameters
+#
+
+class profile::datadog
+{
+    include datadog_agent
```

Comment:
I'm ok with this if we are doing other things, but since it's just a simple include does it make more sense to just include this agent this way as opposed to putting a one liner in the profile?  If you think we'll need to do other things or handle defaults different this makes great sense.  If we are going to do everything in hiera this feels a little superfluous.